src/tcgwars/logic/impl/gen3/Deoxys.groovy

```diff
@@ -1343,6 +1346,7 @@ public enum Deoxys implements LogicCardInfo {
             attackRequirement {}
             onAttack {
               damage 40
+              //TODO: Could this choice be done regardless of having [L] Energy attached or not? May want the self damage for some reason.
```

Comment:
I believe you need to meet the requirement of an "if you do" effect before you can use it. In this case, having an [L] Energy attached.

Reply:
Yeah, it's a weird wording but probably intended the way you're saying. Can't find a single question about this either, so might just leave it like this.